[openhabcloud] Clarify exposing Items (#7446)
openhab/openhab-addons: bundles/org.openhab.io.openhabcloud/README.md

![Configuration](contrib/doc/cfg.png)

Please note, that you should not expose all your items in this settings dialog.

> Also note that at present this feature is currently turned off.

Reviewer: Is it true ?

Reviewer 2: @digitaldan is probably the best one to review/comment here!

Reviewer 3: Right now we are ignoring item updates coming from users by not persisting them to the cloud DB  until i can either 1) optimize the code to not be so hard on the DB or 2) move to a bigger mongodb server (which we are in the process of doing now, but not sure if its big enough).  So more specifically this feature is disabled on the cloud server.

Reviewer 4: Ok, that's sad ... Do you have any other communication channel about that elsewhere? Because I had some problem this week to add new items to myopenhab (and your comment explain verything ...).
So, because I wasn't aware of this problem, I try everything on my system in order to fix something that was disabled on the server side ... (And of course ... I also erase all my devices from myopenhab.org ... thinking it will fix something I did ... So now it's worse ...) now all of my  items/rules are not working ... do you have any workarround for linking Openhab and IFTTT ?

By the way, I have some Java and MongoDB skills if needed.

Author: There is a post on the forum explaining how to set up your own instance of OCS and link it with IFTTT. https://community.openhab.org/t/how-to-integrate-custom-openhab-cloud-with-ifttt/79237

There was an announcement on the forum and several posts about it and there is an open issue (last time I checked) on the repo. But the docs never were updated, in my case, under the assumption that it would be fixed quickly. Since it wasn't fixed quickly (understandably given what the fix involves) I see a steady stream of users like @ozirissp who run into all sorts of trouble. Even worse are those Alexa and GA users who try to expose their Items for that integration needlessly, accomplishing nothing beyond exacerbating the problem which caused it to be turned off in the first place. My thought was to add the warning about not exposing Items for Alexa and GA and that it's currently turned off, and then when it gets turned back on to update the docs again.

You just need to expose those items, which you want to be accessible by IFTTT.

> You should not expose items in this way for Alexa or Google Assistant.

Reviewer 3: ```suggestion
This setting has no affect on Alexa or Google Assistant functionality.
```
